cleanup_all_covid_calls.py

```diff
@@ -0,0 +1,155 @@
+import pandas as pd
+from uszipcode import SearchEngine
+import numpy as np
+from datetime import datetime
+
+file = "Data from 4.2.20 Fake Data.xlsx"
+
+# read all sheets, returns a dict of dataframes
+dfs = pd.read_excel(file, sheet_name=None)
+
+### Cleanup for All COVID Calls dashboard
+
+# step 1
+# select required columns from VIA LINK’s Disaster Form
+# pretty sure the distaster form is "Uncleaned data type 1 VIA LINK"
+VIA_LINK_REQUIRED_COLUMNS_DISASTER = [
+    "CallReportNum",
+    "ReportVersion",
+    "CallDateAndTimeStart",
+    "CityName",
+    "CountyName",
+    "StateProvince",
+    "PostalCode",
+    "Client Information - Age Group",
+    "Client Information - Call Type",
+    "Client Information - Identifies as",
+    "Concerns/Needs - Concerns/Needs",
+    "Contact Source - Program ",  # ending space is needed
+    "Needs - Basic Needs Requested",
+]
+vialink1_df = dfs["Uncleaned data type 1 VIA LINK"][VIA_LINK_REQUIRED_COLUMNS_DISASTER]
+
+# step 2
+# select required columns from 232-Help’s Disaster Form
+TWO32_HELP_REQUIRED_COLUMNS = [
+    "CallReportNum",
+    "ReportVersion",
+    "CallDateAndTimeStart",
+    "CityName",
+    "CountyName",
+    "StateProvince",
+    "PostalCode",
+    "Client Information - Date of Birth",
+    "Client Information - Call Type",
+    "Call Outcome - What concerns/needs were identified?",
+    "Client Information - Identifies as",
+    "Needs - Basic Needs Requested",
+]
+two32_help_df = dfs["Uncleaned Data from 232-Help"][TWO32_HELP_REQUIRED_COLUMNS]
+
+# step 3
+# Create age ranges from date of birth
+# use ranges 0-5, 6-12, 13-17, 18-24, 25-40, 41-59, 60+.
```

Comment:
I'd extract steps like this into a function that you can apply in a pipeline. Something like [what's described here](https://chrisalbon.com/python/data_wrangling/pandas_create_pipeline/).

Reply:
I saw later you mentioned extracting the zipcode stuff into a utils library. That's a good idea, and I'd also create a steps library that has functions that apply each step.

Then you can use each step in the pipeline in a main `start` or `process` function in the main script that's called from the after processing CLI arguments. Something like [click](https://click.palletsprojects.com/en/7.x/) works well for doing the CLI part of that.

Reply:
I'm thinking we can refactor to use pipelines in another PR if needed.  I don't think it's necessary at this point